k8s/deployment.yaml

```
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rdm-dashboard
+  labels:
+    app: rdm-dashboard
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: rdm-dashboard
+  template:
+    metadata:
+      labels:
+        app: rdm-dashboard
+    spec:
+      containers:
+        - name: rdm-dashboard
+          image: ghcr.io/tugraz-rdm/rdm:main
```

```suggestion
          image: ghcr.io/tugraz-rdm/rdm:devel
```
we will stick to the devel  - and branch devel for further developments.

---

Also add a image pull policy : always